core-plugins/docs/HDFSAction-action.md

```diff
@@ -0,0 +1,45 @@
+# HDFS Action
+
+
+Description
+-----------
+Moves file(s) within HDFS.
+
+
+Use Case
+--------
+This action can be used when files needs to be moved to a new location in HDFS for the purpose of archiving,
```

Comment:
Would it make sense to also support the use case of `copy` in this plugin also?
(copy vs move)